doc/book/app_server/cookbook.rst

```diff
@@ -774,7 +774,92 @@ An "HTML" file for this server, including Lua, could look like this
     </body>
     </html>
 
+.. _cookbook-select-all-go:
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+select_all.go
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In Go, there is no one-liner to select all tuples from a Tarantool space.
+Yet you can use a script like this one.
+
+.. code-block:: go
+
+    package main
+
+    import (
+    	"fmt"
+
+    	"github.com/tarantool/go-tarantool"
+    )
+
+    /*
+    box.cfg{listen = 3301}
+    box.schema.user.passwd('pass')
+
+    s = box.schema.space.create('tester')
+    s:format({
+    			{name = 'id', type = 'unsigned'},
+    			{name = 'band_name', type = 'string'},
+    			{name = 'year', type = 'unsigned'}
+    			})
+    s:create_index('primary', {
+    			type = 'hash',
+    			parts = {'id'}
+    			})
+    s:create_index('scanner', { type = 'tree', parts = {'id', 'band_name'} })
+
+    s:insert{1, 'Roxette', 1986}
+    s:insert{2, 'Scorpions', 2015}
+    s:insert{3, 'Ace of Base', 1993}
+    */
+
+    func main() {
+    	conn, err := tarantool.Connect("127.0.0.1:3301", tarantool.Opts{
+    		User: "admin",
```

Comment:
Seems tabs and spaces mixed here. I don't check other places.

Reply:
It's an rst code block, it has 4 spaces before all strings (as I understood).